security/pkg/nodeagent/caclient/client.go

```diff
@@ -54,6 +55,6 @@ func NewCAClient(endpoint, CAProviderName string, tlsFlag bool) (caClientInterfa
 	case googleCA:
 		return gca.NewGoogleCAClient(conn), nil
 	default:
-		return nil, nil
+		return nil, fmt.Errorf("CA provider %q isn't supported, only support %q", CAProviderName, strings.Join([]string{googleCA}, ","))
```

Comment:
Suggest to reword this sentence to something like

"CA provider %q is not supported."

or  

"CA provider %q is not supported. Currently Istio only supports %q."

Reply:
updated.